superset-frontend/src/components/Select/AsyncSelect.tsx

```diff
@@ -574,7 +574,9 @@ const AsyncSelect = forwardRef(
         }
       } else {
         const token = tokenSeparators.find(token => pastedText.includes(token));
-        const array = token ? uniq(pastedText.split(token)) : [pastedText];
+        const array = token
```

Comment:
Even though we can implement this using simple Javascript, `uniq` still offers semantic meaning that helps with readability.